### PR TITLE
7903397: Update changelog for jtreg 7.1.1

### PR DESCRIPTION
**Pull request by sormuras**
Please review this commit that updates the changelog file to include a section for jtreg 7.1.1

It also adds the entry to the list of noteworthy unreleased changes of the upcoming jtreg 7.2 release.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [CODETOOLS-7903397](https://bugs.openjdk.org/browse/CODETOOLS-7903397): Update changelog for jtreg 7.1.1

### Reviewers
 * [Jonathan Gibbons](https://openjdk.org/census#jjg) (@jonathan-gibbons - **Reviewer**) ⚠️ Review applies to [e6649790](https://git.openjdk.org/jtreg/pull/142/files/e6649790be2e3a5458552397683697c8d57d3731)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtreg pull/142/head:pull/142` \
`$ git checkout pull/142`

Update a local copy of the PR: \
`$ git checkout pull/142` \
`$ git pull https://git.openjdk.org/jtreg pull/142/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 142`

View PR using the GUI difftool: \
`$ git pr show -t 142`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtreg/pull/142.diff">https://git.openjdk.org/jtreg/pull/142.diff</a>

</details>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 
 
-## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...master)
+## [Unreleased](https://git.openjdk.org/jtreg/compare/7.1.1+1...master)
```
Comment (jonathan-gibbons) [inline CHANGELOG.md:3]: You dropped the `jtreg-` ... that seems wrong...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 
-_nothing noteworthy, yet_
+* Summary reporter getter is now thread-safe.
+  * [CODETOOLS-7903390](https://bugs.openjdk.org/browse/CODETOOLS-7903390)
+
+## [7.1.1+1](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...7.1.1+1)
```
Comment (jonathan-gibbons) [inline CHANGELOG.md:8]: should there be a `jtreg-` after the `...`

**Review by jonathan-gibbons — CHANGES_REQUESTED** [review]
The tags for 7.1.1 seem incorrect

File: CHANGELOG.md (hunk 1 above)
Comment (sormuras) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1.1+1...master)
 
-_nothing noteworthy, yet_
+* Summary reporter getter is now thread-safe.
+  * [CODETOOLS-7903390](https://bugs.openjdk.org/browse/CODETOOLS-7903390)
+
+## [7.1.1+1](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...jtreg-7.1.1+1)
+
+* Summary reporter getter is now thread-safe.
+  * [CODETOOLS-7903390](https://bugs.openjdk.org/browse/CODETOOLS-7903390)
```
Comment (jonathan-gibbons) [inline CHANGELOG.md:11]: Seems like duplicate entries.

Comment (sormuras) [reply CHANGELOG.md:11]: The intention was to document a backport.

As the upcoming `7.2` release will contain all fixes from older releases, I'll remove the duplicate entry.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,26 @@
 
 
-## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...master)
+## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1.1+1...master)
 
 * Improved support for JUnit Jupiter.
   * Update jtreg to bundle JUnit 5.9.2 [CODETOOLS-7903406](https://bugs.openjdk.org/browse/CODETOOLS-7903406)
+  * Improve reporting for errors in JUnit's lifecycle methods [CODETOOLS-7903430](https://bugs.openjdk.org/browse/CODETOOLS-7903430)
+
+* Improved support TestNG.
```
Comment (jonathan-gibbons) [inline CHANGELOG.md:9]: insert _for_ : Improved support for TestNG

**Review by jonathan-gibbons — APPROVED** [review]
One small grammatical suggestion.

File: CHANGELOG.md (hunk 4 above)
Comment (sormuras) [reply CHANGELOG.md:9]:
```suggestion
* Improved support for TestNG.
```

**Pull request closed without merging**